src/command/identity/export.ts

```diff
@@ -0,0 +1,77 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { exit } from 'process'
+import { Identity, IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+import { printNoIdentitiesError, promptUserForIdentity } from './common'
+
+export class Export extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'export'
+
+  public readonly description = 'Export identity'
+
+  @Argument({ key: 'identity-name', describe: 'Name of the identity to be exported' })
+  public identityName!: string
+
+  public async run(): Promise<void> {
+    this.initCommand()
+    this.checkForNoIdentities()
+    const options = this.getV3Identities()
```

Comment:
I think it would be better to call this `identities` instead of `options`

Reply:
Agreed, renaming done.